test/unit/user_test.rb

```diff
@@ -16,7 +16,7 @@ class UserTest < ActiveSupport::TestCase
   [generate_barcode].each do |barcode|
     test 'sierra user can be found by barcode' do
       mock_check_barcode_request(barcode, '200')
-      response = @user.check_barcode_found_in_sierra(barcode)
+      response = @user.check_barcode_uniqueness_with_sierra(barcode)
       user_would_be_unique_in_sierra = false
```

Comment:
I realise this isn't in your PR, but setting a variable to false and then checking for the negation of it is really hard to read!